sonic-xcvrd/xcvrd/xcvrd_utilities/port_event_helper.py

```diff
@@ -96,6 +109,16 @@ def subscribe_port_update_event(self):
                                             port_tbl, list(d.values())[0], namespace))
         self.sel, self.asic_context = sel, asic_context
 
+    def is_valid_physical_port(self, port_name):
```

Comment:
I don't know if the name of the function is accurate. Why do we use `is_front_panel_port` instead?

Reply:
I actually don't need this method at all. I can just inline the actual call

Reply:
Updated